api.py

```diff
@@ -693,7 +693,53 @@ def create_orchestration(name, orchestration_def, **kwargs):
     return create_json_resource('ORCHESTRATION', name, orchestration_def, **kwargs)
 
 
+def list_api_keys(**kwargs):
+    """
+    Get the users's API keys. 
+
+    Send an HTTP GET request to list this user's API keys.
+
+    Parameters
+    ----------
+    **kwargs
+        See :py:func:`make_get_request`
+
+    Returns
+    -------
+    requests.Response
+        Returns a list of API key dictionaries.
+
+    Raises
+    ------
+    requests.HTTPError
+        Requests that result in an error raise an exception with information about the failure. See :py:func:`requests.Response.raise_for_status` for more information.
+    """
+
+    response = make_get_request('apikeys/list', **kwargs)
+    return json.loads(response.content)
+
+
 def create_api_key(**kwargs):
+    """
+    Add an API key to a user's account.
```

Comment:
supernit: I'd say "Add an API key to *this* user's account."

Reply:
correct -- it's documentation, that stuff is important